target/linux/ipq806x/base-files/etc/hotplug.d/ieee80211/10_fix_wifi_mac

```diff
@@ -9,6 +9,16 @@ PHYNBR=${DEVPATH##*/phy}
 . /lib/functions.sh
 . /lib/functions/system.sh
 
+board=$(board_name)
+
+case "$board" in
+	ubnt,unifi-ap-hd)
+			echo $(macaddr_add $(mtd_get_mac_binary EEPROM 0x6) $(($PHYNBR + 1)) ) > /sys${DEVPATH}/macaddress
+		;;
+	*)
+		;;
```

Comment:
Since I've caught you now with it, would you please tell me what's the point of having an empty default case?

I've been wondering about this all my life...

Reply:
Ok I think you're right, there is no point, just a formality...

I thought it should be as close as possible to what the script originally was...

Reply:
I know of some static code analysis tools that don't like switch statements without default cases, so I stopped discussing when someone asks me to add one when I don't know what tools are used in the background. :) Removed it again.